publisher/product/ontology/build-theallfile.sh

```diff
@@ -32,7 +32,7 @@ function ontologyCreateTheAllTtlFile() {
       --dir="${INPUT}/LCC" \
       --dir=${tag_root} \
       --top="https://spec.edmcouncil.org/${family_product_branch_tag:?}/AboutFIBODev/" \
```

Comment:
Should be:
`--top="https://spec.edmcouncil.org/${family_product_branch_tag:?}/About${ONTPUB_FAMILY^^}Dev/" \`